### PR TITLE
Gitignoreandreadme

### PR DESCRIPTION
**Pull request by sauliuskucevicius**
- Added readme.md
- Added .gitignore
- Removed target folder
- Removed .idea folder

https://ten10solutions.atlassian.net/browse/AUT-32

### CODE REVIEW

File: readme.md
Comment (Capri-sunn) [inline readme.md:1]: This is amazing work Saulius, i trust your work!

**Review by Capri-sunn — APPROVED** [review]
Just amazing work Saulius, fantastic work

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+target/
+.idea/compiler.xml
```
Comment (matthewmccormick) [inline .gitignore]: Could you have this as the whole .idea folder rather than just compiler.xml @sauliuskucevicius ?

**Review by matthewmccormick — DISMISSED** [review]
Requested changes to review on .gitignore

File: .gitignore (hunk 1 above)
Comment (sauliuskucevicius) [reply .gitignore]: Yes - good spot! I will work on those changes now

Comment (sauliuskucevicius) [reply .gitignore]: @matthewmccormick I have made those changes in a commit now, can you review again?

Comment (Capri-sunn) [reply .gitignore]: @matthewmccormick i agree as this helps the various configuration files

Comment (Capri-sunn) [reply .gitignore]: > @matthewmccormick I have made those changes in a commit now, can you review again?

i will review this now instead

**Review by Capri-sunn — APPROVED** [review]
Saulius has made these changes of having the whole .idea folder rather than just the compiler.xml

**Review by BeatrizBravo — APPROVED** [review]
The readme and the .gitignore  are correct

**Comment by BeatrizBravo** [comment]
pull request #9 done.

**Comment by BeatrizBravo** [comment]
there are more unmerged commits and I am going to reopen them to carry on with the merge

**Review by BeatrizBravo — APPROVED** [review]
Approve the merge for redme and gitignore

**Review by squishypandakay — APPROVED** [review]

**Review by Capri-sunn — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (matthewmccormick) [reply .gitignore]: Sorry for the delay, thanks for reviewing 👍

**Pull request merged**